setup.py

```diff
@@ -51,7 +51,6 @@
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
     ],
-    test_suite='tests',
```

Comment:
Is this correct? It made the build fail, and there are no tests yet, so I just removed this line.

Reply:
Yeah removing this is fine for now, eventually there should be tests, but until then we don't need it.